common/logs.py

```diff
@@ -13,10 +13,12 @@
 # limitations under the License.
 """Set up for logging."""
 from enum import Enum
+
 import logging
 import os
 import sys
 import traceback
+import time
```

Comment:
Annoying google nit, can you move this line up so the imports are alphabetical order